content/applications/inventory_and_mrp/barcode/setup/hardware.rst

```diff
@@ -16,12 +16,13 @@ Follow this guide to choose and set up a barcode scanner compatible with Odoo's
 Scanner types
 =============
 
-Before setting up a barcode scanner, it is important to determine which scanner type best meets the
-needs of the business. There are three main types, each with their own benefits and use cases:
+Before setting up a barcode scanner, it is important to determine which type of scanner best meets
+the needs of the business. There are three main types of scanners that can be used with Odoo, each
+with their own benefits and use cases:
 
 - **USB scanners** are connected to a computer, and are suitable for businesses that scan products
-  at a fixed location, like at the checkout in a grocery store. Ensure the chosen USB scanner is
-  compatible with the keyboard layout of the computer.
+  at a fixed location, like at the checkout line in a grocery store. Ensure the chosen USB scanner
+  is compatible with the keyboard layout of the computer.
```

Comment:
```suggestion
  at a fixed location, like at the checkout line in a grocery store.

.. important::
   Ensure the chosen USB scanner is compatible with the keyboard layout of the computer.
```
using this admonition because this information is required in order the user to properly accomplish the goal/use the scanner